DigitalOffice/Services/UserService/Broker/Responses/IInternalUserInfoResponse.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace Broker.Responses
+{
+    /// <summary>
+    /// DTO for dispatch user information through a message broker
+    /// </summary>
+    public interface IInternalUserInfoResponse
+    {
+        public Guid UserId { get; }
+        public string FirstName { get; }
+        public string LastName { get; }
+        public string MiddleName { get; }
+        public string UserPositionName { get; }
+    }
+}
```

Comment:
We use classes for broker communication, but not interface

Reply:
Again. We use interfaces.